garak/generators/huggingface.py

```diff
@@ -127,10 +140,15 @@ def _call_model(
                 i["generated_text"] for i in raw_output
             ]  # generator returns 10 outputs by default in __init__
 
+        if self.use_chat:
+            text_outputs = [_o[-1]["content"].strip() for _o in outputs]
+        else:
+            text_outputs = outputs
+
```

Comment:
Not for this PR - there are multiple ways of representing conversations in garak, and attempt should be canonical for conversation history; I'm starting to consider patterns where attempt holds the conversation history still but where this can be read & written using other interfaces, like an OpenAI API messages dict list, or this HF style format. But mayeb the work is so simple that this current pattern works fine.